.eslintrc

```diff
@@ -2,7 +2,8 @@
   "env": {
     "node": true,
     "browser": true,
-    "mocha": true
+    "mocha": true,
+    "es6": true
```

Comment:
This config option should be added as-needed instead of globally, so ES6 is *blocked* for most files.